packages/styles/scss/components/breadcrumb/_breadcrumb.scss

```diff
@@ -30,13 +30,25 @@
     }
   }
 
+  .#{$prefix}--breadcrumb--sm {
+    @include type-style('label-01');
+  }
+
+  .#{$prefix}--breadcrumb .#{$prefix}--link {
+    font: inherit;
+  }
+
   .#{$prefix}--breadcrumb-item {
     position: relative;
     display: flex;
     align-items: center;
     margin-inline-end: $spacing-03;
   }
 
+  .#{$prefix}--breadcrumb--sm .#{$prefix}--breadcrumb-item {
+    margin-inline-end: $spacing-02;
+  }
```

Comment:
This and the other size-related styling is fine as-is but just as a heads up we'll eventually want this component to take advantage of the contextual layout tokens, https://github.com/carbon-design-system/carbon/issues/13923